src/model_io/urdf/tests/ModelExporterUnitTest.cpp

```diff
@@ -23,6 +23,65 @@
 
 using namespace iDynTree;
 
+unsigned int getNrOfVisuals(const iDynTree::Model& model)
```

Comment:
Is it meaningful to move these methods `getNrOfVisuals()` and `getNrOfCollisions()` to someplace in the library, instead of being in the Tests?

Reply:
To be honest, I am not sure at all they have any use besides this tests. If we find another use outside this either in iDynTree or outside, I will be happy to add it to `iDynTree::Model` or somewhere else, but for the time being I think it make sense to keep it just here.

Reply:
Agreed.